JS/model/homepageModel.js

```diff
@@ -0,0 +1,26 @@
+let homepageModel={
+    topTabCategoryList :[],
+    leftTabCategoryList: {},
+    products: {},
+    
+    setData: function(data) {
+        this.topTabCategoryList = data.topTabCategoryList;
+        this.leftTabCategoryList = data.leftTabCategories;
+        this.products=data.products;
+    },
+    getTopTabCategoriesData: function(){
+        return this.topTabCategoryList;
+    },
+    getLeftTabCategoriesData: function(){
+        return this.leftTabCategoryList;
+    },
+    getLeftTabCategoriesData: function(category){
```

Comment:
Isn't it the same method name already defined on line 14, let's call this `getFilteredLeftTabCategoriesItems`